build/checkstyle/tachyon_checks.xml

```diff
@@ -47,6 +47,11 @@
     <property name="fileExtensions" value=".java"/>
   </module>
   <module name="RegexpSingleline">
+    <property name="format" value="TODO[^\(]"/>
```

Comment:
No, it's used for logging TODOs in checkstyle's output with a specified severity level.
Guys, I think the given rule is good enough for reminding developers to add authors, and it's not necessary to develop a checkstyle custom rule to detect the closing bracket, which requires some advanced RE features.
http://stackoverflow.com/questions/7898310/using-regex-to-balance-match-parenthesis

What do you think?